notebooks/text_classification/solutions/2_keras_for_text_classification.ipynb

```diff
@@ -910,6 +910,218 @@
     "cnn_model.summary()"
```

Comment:
<p>same here. I'd change the PATIENCE to 2 and learning rate to 0.0001, but just to get better plots below. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/81/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='91'/>

Reply:
<p>Done!</p>